libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -1076,6 +1077,10 @@ class Joke(BaseModel):
         models. Must be a string containing the cache name (A sequence of numbers)
     """
 
+    project: Optional[str] = None
```

Comment:
we already have a project_id from the base class, why do we need a separate one?